editoast/src/views/path/path_item_cache.rs

```diff
@@ -97,22 +114,30 @@ impl PathItemCache {
                 PathItemLocation::TrackOffset(track_offset) => {
                     vec![track_offset.clone()]
                 }
-                PathItemLocation::OperationalPointId { operational_point } => {
-                    match self.get_from_id(&operational_point.0) {
-                        Some(op) => op.track_offset(),
-                        None => {
-                            invalid_path_items.push(InvalidPathItem {
-                                index,
-                                path_item: path_item.clone(),
-                            });
-                            continue;
-                        }
+                PathItemLocation::OperationalPointReference(OperationalPointReference {
+                    reference: OperationalPointIdentifier::OperationalPointId { operational_point },
+                    track_reference,
+                }) => match self.get_from_id(&operational_point.0) {
+                    Some(op) => {
+                        let track_offsets = op.track_offset();
+                        self.track_reference_filter(track_offsets, track_reference)
```

Comment:
This does not work since we want to return an error if the filter return an empty list.